### DISCUSSION TITLE
Wind speed reported as null

### DISCUSSION THREAD

**Discussion by sdennstaedt** [weather-gov/api#367] [Q&A]
Looking at the JSON that is returned here the winds are as null.  But METAR suggests 22kts 

https://api.weather.gov/stations/K8D3/observations/latest?require_qc=true

{
    "@context": [
        "https://raw.githubusercontent.com/geojson/geojson-ld/master/contexts/geojson-base.jsonld",
        {
            "wx": "https://api.weather.gov/ontology#",
            "s": "https://schema.org/",
            "geo": "http://www.opengis.net/ont/geosparql#",
            "unit": "http://codes.wmo.int/common/unit/",
            "@vocab": "https://api.weather.gov/ontology#",
            "geometry": {
                "@id": "s:GeoCoordinates",
                "@type": "geo:wktLiteral"
            },
            "city": "s:addressLocality",
            "state": "s:addressRegion",
            "distance": {
                "@id": "s:Distance",
                "@type": "s:QuantitativeValue"
            },
            "bearing": {
                "@type": "s:QuantitativeValue"
            },
            "value": {
                "@id": "s:value"
            },
            "unitCode": {
                "@id": "s:unitCode",
                "@type": "@id"
            },
            "forecastOffice": {
                "@type": "@id"
            },
            "forecastGridData": {
                "@type": "@id"
            },
            "publicZone": {
                "@type": "@id"
            },
            "county": {
                "@type": "@id"
            }
        }
    ],
    "id": "https://api.weather.gov/stations/K8D3/observations/2020-03-08T11:56:00+00:00",
    "type": "Feature",
    "geometry": {
        "type": "Point",
        "coordinates": [
            -96.989999999999995,
            45.670000000000002
        ]
    },
    "properties": {
        "@id": "https://api.weather.gov/stations/K8D3/observations/2020-03-08T11:56:00+00:00",
        "@type": "wx:ObservationStation",
        "elevation": {
            "value": 354,
            "unitCode": "unit:m"
        },
        "station": "https://api.weather.gov/stations/K8D3",
        "timestamp": "2020-03-08T11:56:00+00:00",
        "rawMessage": "K8D3 081156Z AUTO 23022G32KT 07/01 A2958 RMK AO1 PK WND 22034/1111 SLP025 T00720006 10122 20072 55005",
        "textDescription": "Clear",
        "icon": "https://api.weather.gov/icons/land/night/skc?size=medium",
        "presentWeather": [],
        "temperature": {
            "value": 7.2000000000000455,
            "unitCode": "unit:degC",
            "qualityControl": "qc:V"
        },
        "dewpoint": {
            "value": 0.60000000000002274,
            "unitCode": "unit:degC",
            "qualityControl": "qc:V"
        },
        "windDirection": {
            "value": null,
            "unitCode": "unit:degree_(angle)",
            "qualityControl": "qc:Z"
        },
        "windSpeed": {
            "value": null,
            "unitCode": "unit:m_s-1",
            "qualityControl": "qc:Z"
        },
        "windGust": {
            "value": 16.5,
            "unitCode": "unit:m_s-1",
            "qualityControl": "qc:S"
        },
        "barometricPressure": {
            "value": 100170,
            "unitCode": "unit:Pa",
            "qualityControl": "qc:V"
        },
        "seaLevelPressure": {
            "value": 100250,
            "unitCode": "unit:Pa",
            "qualityControl": "qc:V"
        },
        "visibility": {
            "value": null,
            "unitCode": "unit:m",
            "qualityControl": "qc:Z"
        },
        "maxTemperatureLast24Hours": {
            "value": null,
            "unitCode": "unit:degC",
            "qualityControl": null
        },
        "minTemperatureLast24Hours": {
            "value": null,
            "unitCode": "unit:degC",
            "qualityControl": null
        },
        "precipitationLastHour": {
            "value": null,
            "unitCode": "unit:m",
            "qualityControl": "qc:Z"
        },
        "precipitationLast3Hours": {
            "value": null,
            "unitCode": "unit:m",
            "qualityControl": "qc:Z"
        },
        "precipitationLast6Hours": {
            "value": null,
            "unitCode": "unit:m",
            "qualityControl": "qc:Z"
        },
        "relativeHumidity": {
            "value": 62.894657114005227,
            "unitCode": "unit:percent",
            "qualityControl": "qc:C"
        },
        "windChill": {
            "value": null,
            "unitCode": "unit:degC",
            "qualityControl": "qc:V"
        },
        "heatIndex": {
            "value": null,
            "unitCode": "unit:degC",
            "qualityControl": "qc:V"
        },
        "cloudLayers": []
    }
}

**Comment by StephenClouse** [comment]
This is the same root issue as #31 and the same advice applies.

**Reply by zmc** [reply]
@StephenClouse I get nulls constantly, but cannot find issue 31. Would you care to repeat the advice please?

**Reply by scadergit** [reply]
We discontinued issues for discussions (as this board is not for operational issues). Found it on the wayback machine: https://web.archive.org/web/20201029024956/https://github.com/weather-gov/api/issues/31
